scene/shapes.py

```diff
@@ -0,0 +1,45 @@
+import bpy
+import bmesh
+
+def create_sphere(name="new_sphere", size=1.0):
+	'''
+	Creates a sphere with the given size.
+	'''
+	scene = bpy.context.collection
+
+	mesh = bpy.data.meshes.new(name)
+	sphere = bpy.data.objects.new(name, mesh)
+
+	scene.objects.link(sphere)
+
+	bm = bmesh.new()
+	bmesh.ops.create_uvsphere(bm,
+		u_segments=16, v_segments=16,
+		diameter=size
+	)
+	bm.to_mesh(mesh)
+	bm.free()
+
+	return sphere
+
+def create_cone(name="new_cone", base_size=1.0, height=3.0):
```

Comment:
The only difference between `create_sphere` and `create_cone` is the calls `bmesh.ops.create_uvsphere` and `bmesh.ops.create_cone`. Can we do something to reduce the boilerplate required for any new shape?

Reply:
Yeah, but I'd like to get the importing done at a basic level before I go in and do good stuff for shapes. I want to get models in before I worry about those details.

Reply:
I want to move onto markers, and then geometry before I worry too much about the shapes of the nodes and markers